ENH: Support napari layers of corresponding points

N-Dekker opened the pull request:
Allowed the user to specify corresponding points by Points layers.

N-Dekker commented on elastix_napari/elastix_registration.py:
```diff
@@ -3,20 +3,31 @@
+    # The coordinates of the point set returned by point_set_from_points_layer still need to be reversed, manually, when using itk_napari_conversion version 0.5.0.
+    for point in tuple_of_points:
+        std_vector.push_back((point[1], point[0]))
+    return vector_container
```
@thewtex It appears necessary to reverse the coordinates `point[1]` and `point[0]`, for each point from `point_set_from_points_layer`, before passing the points to `itk.elastix_registration_method`. As I mentioned at https://github.com/InsightSoftwareConsortium/itk-napari-conversion/pull/13#discussion_r2560782854

Should this still be fixed in `itk-napari-conversion`?

thewtex replied:
@N-Dekker yes, PR at https://github.com/InsightSoftwareConsortium/itk-napari-conversion/pull/15

N-Dekker replied:
Tested locally, it works 💯 ! Thanks Matt!

N-Dekker committed:
ENH: Support napari layers of corresponding points

Allowed the user to specify corresponding points by Points layers.

N-Dekker committed:
ENH: Upgrade itk_napari_conversion to 0.5.1

Including pull request https://github.com/InsightSoftwareConsortium/itk-napari-conversion/pull/15 commit https://github.com/InsightSoftwareConsortium/itk-napari-conversion/commit/d5e5a73f42ac7ddbe86986e4111b23dc80fcfbfe "Add reverse_coords option to point conversion functions"
